ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-x.y.yaml

```diff
@@ -125,15 +125,6 @@ tests:
       COLLECTION_METHOD: core_bpf
     workflow: stackrox-automation-flavors-eks-e2e
   timeout: 5h0m0s
-- always_run: false
-  as: eks-ebpf-qa-e2e-tests
-  optional: true
-  steps:
-    env:
-      COLLECTION_METHOD: ebpf
-      TEST_SUITE: eks-qa-e2e-tests
```

Comment:
The non-ebpf version does not have TEST_SUITE: eks-qa-e2e-tests. Perhaps that should be added.

Reply:
Replied in <https://github.com/openshift/release/pull/64474#discussion_r2073247359>. Here it is similar.